Gemfile

```diff
@@ -58,3 +58,5 @@ gem 'net-pop'
 gem 'net-smtp'
 
 gem 'carrierwave'
+gem 'i18n_generators'
```

Comment:
翻訳ファイルの生成のためのgemですよね。常に必要なものではないですし、`development`グループに限定しましょう。公式のREADMEにも言及があります。もし読んでないなら、使ったこと無いgemを使うときは公式のREADMEには目を通すようにしたほうがいいですね。

https://github.com/amatsuda/i18n_generators#installation
> Add 'i18n_generators' to your Gemfile, then bundle. Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.

と、言及されてます。

Reply:
READMEは読んでいませんでした。しっかり書いてありましたね。
次回から読むようにします、ありがとうございます。